### PR TITLE
Issue/1502 product image galleries 2

### LINKED ISSUE #1502: Support product image galleries
Right now product detail shows a single product image, even when it has more images. We need to add support for showing all the images.

detail

### PR DESCRIPTION
**Pull request by nbradbury**
Closes #1502 - adds support for showing all product images instead of just the first one.

multi

If there's only one photo, we show it centered in the detail screen.

single

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/push/FCMRegistrationIntentService.kt [hunk 1]
```diff
@@ -2,8 +2,8 @@ package com.woocommerce.android.push
 
 import android.content.Context
 import android.content.Intent
-import androidx.preference.PreferenceManager
 import androidx.core.app.JobIntentService
+import androidx.preference.PreferenceManager
```
Comment (AmandaRiu) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/push/FCMRegistrationIntentService.kt]: np: Did you mean to check in these changes? They don't seem to be related.

**Review by AmandaRiu — COMMENTED** [review]
Code changes look good @nbradbury ! I did find an unrelated issue with grouped products and opened a separate ticket for that (#1516 ). I left a really low priority code comment (more just a question). 

Also, I noticed that when the user clicks to open the product image, they can't just swipe to view the next image. They have to manually close the image and click on the next one which isn't ideal. I recommend implementing a way for the user to swipe through the images when they have them opened into the viewer. WDYT?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/push/FCMRegistrationIntentService.kt (hunk 1 above)
Comment (nbradbury) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/push/FCMRegistrationIntentService.kt]: I originally branched this PR from my image chooser PR since both were part of the add/edit products project, but then we decided the image gallery should go into `develop` so I removed all the image chooser code. As a result there are a few minor changes here that aren't related to the image gallery. Sorry for the confusion!

**Comment by nbradbury** [comment]
> I recommend implementing a way for the user to swipe through the images when they have them opened into the viewer. WDYT?

Yes, definitely! That's no minor task, though, so I've added issue #1517 to address this separately.

**Comment by kyleaparker** [comment]
Images should be aligned on the left with the text, and if there is one image it should also be left aligned with the text

**Comment by nbradbury** [comment]
> Images should be aligned on the left with the text, and if there is one image it should also be left aligned with the text

@kyleaparker Thanks for the review! I've fixed both of those issues.

multi

single

**Review by AmandaRiu — APPROVED** [review]
Verified design changes. LGTM! :shipit:
